_resources/port1.0/group/qmake5-1.0.tcl

```diff
@@ -54,6 +54,10 @@ pre-configure {
         if { ![file exists ${sdks_dir}/MacOSX${configure.sdk_version}.sdk] } {
             configure.sdk_version
         }
+
+        if {${developer_dir} ne ${configure.developer_dir} && ![file exists ${configure.developer_dir}/SDKs/MacOSX${configure.sdk_version}.sdk]} {
```

Comment:
`configure.developer_dir` doesn't exist on a released version, so we need to add a `info exists` check here.